### ISSUE TITLE
Error runtime

### ISSUE THREAD

**Issue by amarye** [arneschneuing/DiffSBDD#22]
When generate_ligands.py was running, I got an error.

Cloud you please help me to solve this problem?

![diffsbdd](https://user-images.githubusercontent.com/55770700/236127291-9c3d161d-b002-441a-9be7-53b9e5e3bcd2.jpg)

**Comment by msako82** [comment]
I got the same error too.

I deleted the .cpu() in lines 809-810 of lightning_modules.py and the errors disappeared.

        # Build mol objects
        x = xh_lig[:, :self.x_dims].detach().cpu()
        atom_type = xh_lig[:, self.x_dims:].argmax(1).detach().cpu()

I changed the above as follows

        # Build mol objects
        x = xh_lig[:, :self.x_dims].detach()
        atom_type = xh_lig[:, self.x_dims:].argmax(1).detach()

I changed it as follows.

**Comment by amarye** [comment]
Thank you @msako82. I changed the line in Jupyter Notebook 
from 
'device = 'cuda' if torch.cuda.is_available() else 'cpu'' 
to 
'device = torch.device("cpu")'.   
It worked for me.

**Comment by arneschneuing** [comment]
Hello, thanks for your answers!
Interestingly, this error didn't occur in my environment but this [commit](https://github.com/arneschneuing/DiffSBDD/commit/0a142c8f69e1a4f4e3882b37c05ebad3bb36909f) should fix the issue for everyone.

**Comment by Jennyhu666** [comment]
> Thank you [@msako82](https://github.com/msako82). I changed the line in Jupyter Notebook from 'device = 'cuda' if torch.cuda.is_available() else 'cpu'' to 'device = torch.device("cpu")'. It worked for me.

Hello, I encountered the same problem when I was running the inpaint.py and optimized.py scripts, and it was perfectly resolved using the aforementioned method. However I have a question: Will setting device = torch.device("cpu") affect the running speed of the software? Because I checked the GPU occupancy rate through nvidia-smi and it was very low.
